### PR TITLE
Documentation updates: usage tracking, vanilla Kubeflow and more

### PR DESCRIPTION
**Pull request by surajkota**
**Which issue is resolved by this Pull Request:**
- TODO item from #92 regarding detailed documentation for telemetry component
- Moved the installation instructions for vanilla Kubeflow to the vanilla directory and added cluster creation steps for easy reference on Kubeflow.org website is working on
- Changed the name from AWS distribution of Kubeflow to Kubeflow on AWS to be consistent with website and usage tracking documentation
- Added a section in vanilla Kubeflow readme: `Exposing Kubeflow over Load Balancer` to this [#67 (comment)](https://github.com/awslabs/kubeflow-manifests/issues/67#issuecomment-1032128251) to expose deployment over LoadBalancer.

**Testing**
- [x] links are working as expected

### CODE REVIEW

**Review by rrrkharse — APPROVED** [review]

**Review by ryansteakley — APPROVED** [review]

**Review by AlexandreBrown — APPROVED** [review]

**Comment by surajkota** [comment]
Fixed all the broken links, please take another look

File: distributions/aws/examples/vanilla/README.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+
+## Prerequisites
+
+This guide assumes that you have:
+
+1. Installed the following tools on the client machine
+    - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) - A command line tool for interacting with AWS services.
+    - [eksctl](https://eksctl.io/introduction/#installation) - A command line tool for working with EKS clusters.
+    - [kubectl](https://kubernetes.io/docs/tasks/tools) - A command line tool for working with Kubernetes clusters.
+    - [yq](https://mikefarah.gitbook.io/yq) - A command line tool for YAML processing. (For Linux environments, use the [wget plain binary installation](https://mikefarah.gitbook.io/yq/#wget))
```
Comment (ryansteakley) [inline distributions/aws/examples/vanilla/README.md:13]: Link doesn’t seem to be accurate and suggests going to GitHub page for wget https://github.com/mikefarah/yq/#wget

**Review by rrrkharse — APPROVED** [review]
Clicked on all the links, and tried to follow the path of a first time user as much as I could without bias of knowing the repository. The overall pathing is much improved and links work.

**Review by ryansteakley — APPROVED** [review]

**Comment by ryansteakley** [comment]
Good work :)

**Pull request merged**